src/legacy/core_plugins/kibana/public/doc_viewer/_doc_viewer.scss

```diff
@@ -28,15 +37,17 @@
   }
 }
 
-.kbnDocViewer__content {
-  background-color: $euiColorEmptyShade;
-  padding: $euiSizeS 0 0 0;
-}
-
 .kbnDocViewer__buttons,
 .kbnDocViewer__field {
   white-space: nowrap;
 }
+.kbnDocViewer__buttons {
+  width: 60px;
+}
+
+.kbnDocViewer__field {
+  width: 160px;
+}
 
 .kbnDocViewer__actionButton {
```

Comment:
Really small thing, but the way the buttons are hidden is causing them to "stay" when you have clicked one and navigate away with the mouse which looks kinda strange:
<img width="598" alt="Screenshot 2019-08-26 at 09 41 05" src="https://user-images.githubusercontent.com/1508364/63674017-e1b9d000-c7e5-11e9-8cda-4bb0383d8277.png">

I filtered for present on continent name and then navigated away to the gender row, but the present filter stays focussed and thus visible. Initially I thought this happened on purpose and was some kind of state (if the filter is active, the button is shown all the time() and was confused it disappeared when doing something else.

Reply:
agreed, removed the opacity:1 of the elements focus state 
FYI: @cchaos & @ryankeairns